<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTextEdit" name="text_mess" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>40</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btOpen" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>270</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>OpenPort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="l_port" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>230</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDelete" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>270</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>DeletePort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_search" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>10</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search Device</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuitter" >
   <property name="text" >
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionJoin" >
   <property name="text" >
    <string>join</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
